ads/google/imaVideo.js

```diff
@@ -283,6 +283,17 @@ export function imaVideo(global, data) {
   });
   controlsVisible = false;
 
+  // Play button
```

Comment:
Moves play button before the ad countdown timer so that the ad button is in first position whether the ad is playing or the video is playing